ansible/roles/operator-pipeline/templates/openshift/tasks/get-supported-versions.yml

```diff
@@ -48,11 +48,13 @@ spec:
         echo $VERSION_INFO \
           | jq -r '[.indices[].path]' \
           | tr -d '\n\r' \
+          | base64 \
           | tee $(results.indices.path)
 
         echo $VERSION_INFO \
           | jq -r '[.indices[].ocp_version]' \
           | tr -d '\n\r' \
+          | base64 \
```

Comment:
Please include a comment about this workaround with any related links to the details. It would also be helpful to mention the base64 encoding in the description of these fields.

Reply:
Yes, that was the plan. I just finished reporting the bug upstream. I'll add a comment with a link to the gh issue.